src/app/pages/accounts/reset-password/reset-password.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="container">
+    <biosys-header></biosys-header>
+    <ng-container *ngIf="!isPasswordReset; else passwordReset">
+        <div class="row mt-3">
+            <div class="col">
+                <h4>Reset your password</h4>
+                <p>Please enter your new password in the fields below:</p>
+            </div>
+        </div>
+        <div class="row mt-1 main-section">
+            <div class="col">
+                <form [formGroup]="resetPasswordForm" (submit)="resetPassword()">
+                    <div class="form-group row">
+                        <label for="password" class="col-2">Password</label>
+                        <input id="password" class="col-3" type="password" pInputText formControlName="password"/>
+                    </div>
+                    <div class="form-group row">
+                        <label for="passwordConfirm" class="col-2">Password (confirm)</label>
+                        <input id="passwordConfirm" class="col-3" type="password" pInputText
+                               formControlName="passwordConfirm"/>
+                    </div>
+                    <div class="form-group row">
+                        <div class="col-5" style="padding-right: 0px">
+                            <button pButton style="margin-right: 0" class="float-right" type="submit"
+                                    label="Reset Password" [disabled]="resetPasswordForm.invalid">
+                            </button>
+                        </div>
+                    </div>
+                </form>
+            </div>
+        </div>
+    </ng-container>
+    <ng-template #passwordReset>
```

Comment:
I'm a bit confused about these `<ng-container>` vs `<ng-template>`. I've got some reading to do.